internal/pkg/service/buffer/worker/service/file_import.go

```diff
@@ -51,7 +52,33 @@ func (s *Service) importFiles(ctx context.Context, wg *sync.WaitGroup, d depende
 					}
 				}()
 
-				// Skip empty?
+				// Skip empty
+				if fileRes.IsEmpty {
+					// Load token
+					token, err := s.store.GetToken(ctx, fileRes.ExportKey)
+					if err != nil {
+						return "", errors.Errorf(`cannot load token for export "%s": %w`, fileRes.ExportKey, err)
+					}
+
```

Comment:
Ak je subor prazdny, tak sa vramci `import` operacie zmaze resource a subor sa oznaci ako `imported`.

Obsahuje `IsEmpty=true`, ak by sme to este na nieco potrebovali.